### DISCUSSION TITLE
Add GitHub profile stats

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#67706] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

At the moment you can simply see contributions per year and can optionally include private contributions.
It would be nice if one can additionally see stats (with the option to include private) like total contributions, total commits, total PRs, total issues, most used languages

There are some services which create GitHub stats for you which you can then place in your profile readme, but these stats are a) not very reliable (I have them and at the moment 1 of 2 is not available = hidden; yesterday, the other one was not available) and b) they can't access private data.
Even if I want to simply show stats for public contributions, it would be better if GitHub could provide reliable data.

These stats can be interesting for one personally but also for recruiters.

Example from https://github.com/anuraghazra/github-readme-stats
![image](https://github.com/community/community/assets/20150243/23f745a4-7b5e-4eb4-a696-4fcaef03b7c6)

![image](https://github.com/community/community/assets/20150243/2e9775b9-8481-4a08-b17e-918c1208e7ab)

**Comment by spenserblack** [comment]
I'd love to have the "most used languages" stat builtin to my profile. I've used both anuraghazra/github-readme-stats and lowlighter/metrics, but they aren't perfect. The former gets rate-limited often, and the latter doesn't seem 100% consistent.
